SabinePfanner/plants-app
# Feature filter period

To test this feature:

- Add a plant by clicking on the '+' button on the homepage.
- Log in using credentials 'crops'/'crops'.
- Fill in all entries and set periods by using the custom timeline inputs at the bottom. This should work by either mouse-clicking or by tab-navigation (shift-tab: move backwards) and pressing Enter. To test grouping of periods into active/non-active, have one or more periods include the current time interval (mid June) (and, vice versa, one or more should exclude it).
- Click 'save', periods should be displayed at the bottom of the plant details page you're routed to.
- On the home page, there is now a filter 'Current Tasks' allowing you to filter our plants that have task periods that are currently relevant.
